Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org/'
+
+# for publishing a new release
+gem 'travis', '1.8.9'
```

Comment:
`travis` gem is required if you want to change the deployment. It's not necessary to publish a new release.